plugins/modules/ecs_service.py

```diff
@@ -683,6 +691,7 @@ def main():
             assign_public_ip=dict(type='bool')
         )),
         launch_type=dict(required=False, choices=['EC2', 'FARGATE']),
+        platform_version=dict(required=False, choices=['1.0.0', '1.1.0', '1.2.0', '1.3.0', '1.4.0', 'LATEST']),
```

Comment:
I have a nasty feeling that hard coding a list of versions here will mean we need to keep updating this.  Is there a way to pull a list of available versions so that you could pre-validate the data provided here?

Reply:
@tremble 's comment is especially germane since AWS practice is to NOT bump "LATEST" to the actual latest platform version.  Every time there is a new platform version, it will be impossible to access it without a corresponding hardcoded module update.

But why not skip validations and just require the module user to know what versions are valid?  Anyone not wanting to use defaults should know what is needed.